content/en/docs/tasks/administer-cluster/coredns.md

```diff
@@ -19,32 +16,28 @@ discovery.
 
 {{% capture steps %}}
 
-## Installing CoreDNS with kubeadm
-
-In Kubernetes 1.9, [CoreDNS](https://coredns.io) is available as an alpha feature, and
-in Kubernetes 1.10 it is available as a beta feature. In either case, you may install
-it during cluster creation by setting the `CoreDNS` feature gate to `true` during `kubeadm init`:
+## Installing kube-dns with kubeadm
 
+In Kubernetes 1.11, [CoreDNS](https://coredns.io) has graduated to General Availability (GA)
+and is installed by default. To install kube-dns instead, set the `CoreDNS` feature gate
+value to `false`:
 ```
-kubeadm init --feature-gates=CoreDNS=true
+kubeadm init --feature-gates=CoreDNS=false
 ```
 
-This installs CoreDNS instead of kube-dns.
-
 ## Upgrading an Existing Cluster with kubeadm
 
-In Kubernetes 1.10, you can also move to CoreDNS when you use `kubeadm` to upgrade
+In Kubernetes version 1.10 and later, you can also move to CoreDNS when you use `kubeadm` to upgrade
 a cluster that is using `kube-dns`. In this case, `kubeadm` will generate the CoreDNS configuration
```

Comment:
Can you add a line to clarify that we need to set the CoreDNS feature-gates to true
 `kubeadm upgrade apply v1.11.0 --feature-gates=CoreDNS=true` when moving from kube-dns to CoreDNS during an upgrade.

Reply:
sure.

Reply:
Is this done?

Reply:
yes.